index.bs

```diff
@@ -3427,9 +3427,9 @@ Maintenance Without a Group</h4>
 <h4 id="contributor-license">
 License Grants from Non-Participants</h4>
 
-	When a party who is not already obligated under the Patent Policy
-	offers a change in class 3 or 4
+	When a proposal for a change in class 3 or 4
 	(as described in [[#correction-classes]]) to a technical report under this process
+	originates from or contains substantive contributions by a party not already obligated under the patent Policy,
```

Comment:
This gives rise to a potential edge case that some change originates from _more than one_ party not already obligated under the patent Policy.

If I'm reading it correctly, under this scenario, the Team can satisfy this changed Process requirement by getting a commitment from only one of them, rather than all of them.